make_index.py

```diff
@@ -48,6 +52,8 @@ def _make_info(filename: str) -> dict[str, Any]:
     return {
         "filename": os.path.basename(filename),
         "hash": f"sha256={sha256}",
+        "core_metadata": f"sha256={metadata_sha256}",
+        "_metadata": metadata_bytes,
```

Comment:
we shouldn't be smuggling unrelated things in a dictionary.  this is intended to be exactly the structure dumb-pypi generates